sklearn/model_selection/_validation.py

```diff
@@ -545,8 +545,9 @@ def permutation_test_score(estimator, X, y, groups=None, cv=None,
         supervised learning.
 
     groups : array-like, with shape (n_samples,), optional
-        Group labels used to constrain the permutation to specific subsets of
-        data.
+        Labels to constrain permutation within groups, i.e. ``y`` values
+        are permuted among samples with the same group identifier.
+        When not specified, ``y`` values are permuted among all samples.
```

Comment:
Ah, sorry. Still not used to github's "viewing a subset of changes". I don't see why this is a NOTE as distinct from a parameter description, though.